__tests__/app.test.js

```diff
@@ -216,4 +216,63 @@ describe("GET /api/articles/:article_id", () => {
         expect(error).toBe("bad request");
       });
   });
+
+  test("404: responds with an error when a non-existant article_id given", () => {
+    return request(app)
+      .get("/api/articles/898989")
+      .expect(404)
+      .then((response) => {
+        const error = response.body.msg;
+        expect(error).toBe("not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("202: adds the amount of votes in the given votes object to the total votes of given article_id", () => {
```

Comment:
`200`, 202 suggests the response is okay but the process incomplete

Reply:
Thanks for the feedback, added the test for negative vote value error and also changed response code to 200 for the success test on line 231 of app test.